app/src/main/java/com/itsaky/androidide/localWebServer/WebServer.kt

```
@@ -365,4 +516,158 @@ WHERE  path = ?
+            // Add 1 to substring start index to handle the "=" between the POST data
+            // field name and the POST data field data.
+            decoded.substring(postDataFieldName.length + 1)
+        } else{
+            log.warn("Expecting a data field named " + postDataFieldName)
+            ""
+        }
+
+        val file = File(playgroundFilePath)
+        try {
+            file.writeText(inputAsString)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)